src/System.Globalization.Calendars/tests/System.Globalization.Calendars.Tests.csproj

```diff
@@ -0,0 +1,118 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{9574CEEC-5554-411B-B44C-6CA9EC1CEB08}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AssemblyName>System.Globalization.Calendars.Tests</AssemblyName>
+    <RootNamespace>System.Globalization.Calendars.Tests</RootNamespace>
+  </PropertyGroup>
```

Comment:
Although amazingly it looks like only a few of the many tests failed in the CI run on Linux:
```
System.Globalization.Calendars.Tests  Total: 566, Errors: 0, Failed: 10, Skipped: 0, Time: 1.477s
```
Given how few there are, it may be better to just suppress those tests for now by attributing them with:
```
[ActiveIssue(846, PlatformID.AnyUnix)]
```
rather than suppressing the entire library.